pkg/api/api.go

```diff
@@ -174,6 +176,13 @@ func (a *API) RegisterAPI(httpPathPrefix string, cfg interface{}) {
 	a.RegisterRoute("/debug/fgprof", fgprof.Handler(), false, "GET")
 }
 
+// RegisterRuntimeConfig registers the endpoints associates with the runtime configuration
+func (a *API) RegisterRuntimeConfig(runtimeCfgManager *runtimeconfig.Manager) {
+	a.indexPage.AddLink(SectionAdminEndpoints, "/runtime_config", "Current Runtime Config")
```

Comment:
I'd mention it includes overrides too: "Current Runtime Config (incl. Overrides)"